coreutils/Makefile

```diff
@@ -36,9 +36,16 @@ include ../Makefile.targ
 AUTOCONF_ENV.64 += FORCE_UNSAFE_CONFIGURE=1
 OVERRIDES.64 += FORCE_UNSAFE_CONFIGURE=1
 
+LD_LIBRARY_PATH_HACK = /tmp/extra-ldlibpath
+PATCHES = Patches/*
+
 all: all_autoconf
 
-install: all
+scribble_ld_library:
+	echo $(DESTDIR)/lib/64 > $(LD_LIBRARY_PATH_HACK)
```

Comment:
... it gets populated here.

To not mess with GNU coreutils compilation environments all environment variables associated with target directories get wiped out (we do the copying by hand).

This hack (and it is a hack) allows the change in coreutils' `Makefile.in` to figure out where to set the LD_LIBRARY_PATH to allow `help2man` invocations of freshly-compiled binaries to actually run.

This exposes IMHO a big downside of the GNU-or-Linux philosophy of `$CMD --help` is the manual page.  🙃